defaults/main.yml

```diff
@@ -8,8 +8,8 @@ hidden_service_services:
     hidden_service_hostname:
     hidden_service_ports:
       - [22, 22]
+    hidden_service_authorized_clients: []
 
 hidden_services_configuration:
   SocksPort: 9050
   SocksPolicy: "reject *"
-
```

Comment:
Automatic linting, I can revert if you feel strongly about trailing empty lines (and want them). :smile: